src/PlantBundle/DataFixtures/ORM/LoadPlantData.php

```diff
@@ -102,6 +102,90 @@ public function load(ObjectManager $manager)
 
         $manager->flush();
 
+        /**
+         * @var User $user
+         */
+        $user = $this->getReference('admin-user');
+
+        $plant = new Plant();
+        $plant->setName("Azalia");;
```

Comment:
This seems like good idea to chain those methods. You do not have to constantly type `$plant->`, only arrow is necessary.
```
$plant = new Plant();
    ->setName("Azalia")
    ->setDescription(...)
    ->setFrequency(...);
```